lib/active_admin/resource_controller/forms.rb

```diff
@@ -0,0 +1,25 @@
+module ActiveAdmin
+  class ResourceController < BaseController
+    module Forms
+
+    protected
+
+      def apply_form(resource)
+        apply_form? ? form_class.new(resource) : resource
+      end
+
+    private
+
+      def apply_form?
+        case action_name
+        when 'new', 'edit', 'update', 'create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.